core/python_utils.py

```diff
@@ -104,28 +104,22 @@ def redirect_stdout(new_target):
     return impl(new_target)
 
 
-def get_args_of_function_node(function_node, args_to_ignore):
-    """Extracts the arguments from a function definition.
+def string_io(buffer_value=''):
+    """Returns StringIO from StringIO module if run under Python 2 and from io
```

Comment:
Why are we adding this function?

Reply:
@DubeySandeep This method was added in one of the merges when I updated with the oppia:develop branch?  should we keep this or remove it?

Reply:
I think we are not expecting to add these code changes through this PR, right? So let's not add it! (In general, the PR should only have the changes you are intending/expecting to add!)